assets/js/blocks/cart/inner-blocks/cart-cross-sells-products/block.json

```
@@ -0,0 +1,30 @@
+{
+	"name": "woocommerce/cart-cross-sells-products-block",
+	"version": "1.0.0",
+	"title": "Cart Cross-Sells products",
+	"description": "Shows the Cross-Sells products.",
+	"category": "woocommerce",
+	"supports": {
+		"align": false,
+		"html": false,
+		"multiple": false,
+		"reusable": false,
+		"inserter": false
+	},
+	"attributes": {
+		"columns": {
+			"type": "number",
+			"default": 3
+		},
+		"lock": {
+			"type": "object",
+			"default": {
+				"remove": true,
+				"move": true
```

If you prefer we can let the user move the `Cart Cross-Sells products` inner block even if it's locked using the following change:

```suggestion
				"move": false
```

---

Thanks for suggesting this. As the Cross-Sells products are currently located below the cart items, I'd leave the move lock active, to prevent that a merchant accidentally places the Cross-Sells block above the cart items.